spring-cloud-starter-task-composedtaskrunner/src/main/java/org/springframework/cloud/task/app/composedtaskrunner/DataFlowConfiguration.java

```diff
@@ -55,7 +56,14 @@ public TaskOperations taskOperations(DataFlowOperations dataFlowOperations) {
 	public DataFlowOperations dataFlowOperations() {
 		final RestTemplate restTemplate = DataFlowTemplate.getDefaultDataflowRestTemplate();
 		validateUsernamePassword(this.properties.getDataflowServerUsername(), this.properties.getDataflowServerPassword());
-		if (StringUtils.hasText(this.properties.getDataflowServerUsername())
+		if (StringUtils.hasText(this.properties.getDataflowServerAccessToken())) {
+			restTemplate.setRequestFactory(
+				HttpClientConfigurer.create(this.properties.getDataflowServerUri()).buildClientHttpRequestFactory()
+			);
+			restTemplate.getInterceptors().add(new OAuth2AccessTokenProvidingClientHttpRequestInterceptor(this.properties.getDataflowServerAccessToken()));
```

Comment:
Its failing to build because this class is no longer present in SCDF.

Reply:
Please make sure you have built and installed https://github.com/spring-cloud/spring-cloud-common-security-config/pull/60